metricbeat/module/kubernetes/pod/data.go

```diff
@@ -130,6 +130,24 @@ func eventMapping(content []byte, perfMetrics *util.PerfMetricsCache) ([]common.
 			podEvent.Put("memory.usage.limit.pct", float64(usageMem)/memLimit)
 		}
 
+		if usageMem > 0 {
+			if nodeMem > 0 {
+				podEvent.Put("memory.usage.node.pct", float64(usageMem)/nodeMem)
+			}
+			if memLimit > 0 {
+				podEvent.Put("memory.usage.limit.pct", float64(usageMem)/memLimit)
+			}
```

Comment:
Please remove the previous code used to calculate these pcts some lines above.

Reply:
Done!